package.json

```diff
@@ -1,17 +1,24 @@
 {
   "name": "simplr-logger",
-  "version": "0.3.0",
-  "description": "Simple JavaScript logger written in TypeScript.",
-  "main": "dist/simplr-logger.js",
+  "version": "0.4.0-alpha.6",
+  "description": "Simple JavaScript logger written in TypeScript. It's can be used in browser or node application.",
```

Comment:
"Simple JavaScript logger written in TypeScript that can be used in browser or node application." or 
"Simple JavaScript logger written in TypeScript. It can be used in browser or node application."